upgrading/migrating_etcd.adoc

```diff
@@ -137,14 +137,26 @@ At the same time, all etcd members and master daemons must be stopped.
 To check the etcd cluster is healthy you can run:
 +
 ----
+# export ETCDCTL_API=3
 # etcdctl <certificate_details> <endpoint> cluster-health <1>
 member 2a3d833935d9d076 is healthy: got healthy result from https://etcd-test-1:2379
 member a83a3258059fee18 is healthy: got healthy result from https://etcd-test-2:2379
 member 22a9f2ddf18fee5f is healthy: got healthy result from https://etcd-test-3:2379
 cluster is healthy
 ----
-<1> For `<certificate_details>`, see
-xref:../admin_guide/backup_restore.adoc#adding-addtl-etcd-members[Backup and Restore] for an example of how to set certificate flags.
+<1> For `<certificate_details>`:
++
+v2:
++
+---- 
+ --ca-file=/etc/origin/master/master.etcd-ca.crt --cert-file=/etc/origin/master/master.etcd-client.crt --key-file=/etc/origin/master/master.etcd-client.key
+----
++
+v3:
++
+----
+--cacert="/etc/origin/master/master.etcd-ca.crt" --cert="/etc/origin/master/master.etcd-client.crt" --key="/etc/origin/master/master.etcd-client.key"
+----
```

Comment:
I'd just provide the full commands, the certificate information is going to be the same for every cluster. The endpoints which requires `--endpoints` flag will vary though. Also, since this command is a v2 command and the raft endpoint status below is a v3 command, I think it's easier to just provide the full commands rather than having a callout that confuses v2 and v3 flags.

```
# etcdctl --ca-file=/etc/origin/master/master.etcd-ca.crt  \
--cert-file=/etc/origin/master/master.etcd-client.crt \
--key-file=/etc/origin/master/master.etcd-client.key \
--endpoints https://etcd-test-1:2379,https://etcd-test-2:2379,https://etcd-test-3:2379 \
cluster-health
```

Then for the raft indices, that's a v3 command.
```
# ETCDCTL_API=3 etcdctl --cacert="/etc/origin/master/master.etcd-ca.crt" \
--cert="/etc/origin/master/master.etcd-client.crt" \
--key="/etc/origin/master/master.etcd-client.key" \
--endpoints https://etcd-test-1:2379,https://etcd-test-2:2379,https://etcd-test-3:2379 \
-w table endpoint status
```

Reply:
@sdodson Thank you. I completely missed what Steven was asking for in the BZ. 

Will folks know what the etcd endpoints are?  Should we put:
--endpoints <etcd-endpoint>,<etcd-endpoint>,<etcd-endpoint>

Reply:
Yeah, they should know their endpoints. Fine with changing it per your suggestion so they don't copy/paste and expect it to work without modification. Just need to make sure they know it's list of comma separated URIs.